Apps/W1/HybridGP/app/src/Migration/Accounts/GPAccountMigrator.codeunit.al

```diff
@@ -232,36 +335,68 @@ codeunit 4017 "GP Account Migrator"
         GLAccDataMigrationFacade.ModifyGLAccount(true);
     end;
 
-    procedure GenerateGLTransactionBatches(GPAccount: Record "GP Account");
+    local procedure MigrateStatisticalAccountDetailsImp(GPAccount: Record "GP Account")
+    var
+        StatisticalAccount: Record "Statistical Account";
+        DataMigrationErrorLogging: Codeunit "Data Migration Error Logging";
+        AccountNum: Code[20];
+    begin
+        AccountNum := CopyStr(GPAccount.AcctNum.Trim(), 1, MaxStrLen(StatisticalAccount."No."));
+
+        if StatisticalAccount.Get(AccountNum) then
+            exit;
+
+        DataMigrationErrorLogging.SetLastRecordUnderProcessing(Format(GPAccount.RecordId));
+
+        StatisticalAccount.Init();
```

Comment:
Minor - Init is useless, everything is initialized. It is used only to clear the non-primary key fields. Recommendaiton is to delete

Reply:
Updates, thanks